diff --git a/libs/base/logging.cpp b/libs/base/logging.cpp
index 6b84f4bff..d51560bf8 100644
--- a/libs/base/logging.cpp
+++ b/libs/base/logging.cpp
@@ -33,3 +33,4 @@ std::optional<LogLevel> FromString(std::string const & s)
   auto const & names = GetLogLevelNames();
-  auto const it = std::find(names.begin(), names.end(), std::toupper(s[0]));
+  // Cast to unsigned char to avoid assertion failure on Windows with non-ASCII characters
+  auto const it = std::find(names.begin(), names.end(), std::toupper(static_cast<unsigned char>(s[0])));
   if (it == names.end())
@@ -123,3 +123,8 @@ LogLevel GetDefaultLogAbortLevel()
-AtomicLogLevel g_LogLevel = {GetDefaultLogLevel()};
-AtomicLogLevel g_LogAbortLevel = {GetDefaultLogAbortLevel()};
+#if defined(DEBUG)
+AtomicLogLevel g_LogLevel = ATOMIC_VAR_INIT(LDEBUG);
+AtomicLogLevel g_LogAbortLevel = ATOMIC_VAR_INIT(LERROR);
+#else
+AtomicLogLevel g_LogLevel = ATOMIC_VAR_INIT(LINFO);
+AtomicLogLevel g_LogAbortLevel = ATOMIC_VAR_INIT(LCRITICAL);
+#endif  // defined(DEBUG)
 }  // namespace base
