diff --git a/libs/platform/location_service/CMakeLists.txt b/libs/platform/location_service/CMakeLists.txt
index 3dbafb80e..106a21713 100644
--- a/libs/platform/location_service/CMakeLists.txt
+++ b/libs/platform/location_service/CMakeLists.txt
@@ -1,6 +1,6 @@
 project(location_service)
 
-if (NOT SKIP_QT_GUI AND PLATFORM_DESKTOP AND PLATFORM_LINUX)
+if (NOT SKIP_QT_POSITIONING AND NOT SKIP_QT_GUI AND PLATFORM_DESKTOP AND PLATFORM_LINUX)
   message("Building with Qt Positioning")
   find_package(Qt6 REQUIRED COMPONENTS Positioning)
   set(QT_LOCATION_SERVICE true)
