diff --git a/libs/base/localisation.hpp b/libs/base/localisation.hpp
index 5e917b78c..f7fcb52ce 100644
--- a/libs/base/localisation.hpp
+++ b/libs/base/localisation.hpp
@@ -38,7 +38,7 @@ static LanguageIndex constexpr kKoreanRomanizedLanguageIndex = 4;
 static LanguageIndex constexpr kChineseLanguageIndex = 10;
 static LanguageIndex constexpr kChinesePinyinLanguageIndex = 22;
 static int8_t constexpr kMaxSupportedLanguages = 64;
-static LanguageCode constexpr kReservedLanguageCode = "reserved";
+static char const constexpr kReservedLanguageCode[] = "reserved";
 static string const kMapLanguageSetting = "MapLanguageCode";
 static string const kAlternativeMapLanguageHandlingSetting = "MapLanguageLimitAlternativesToLocal";
 static string const kTransliterationSetting = "TransliterationMode";
